demo/demo.js

```diff
@@ -62,7 +62,9 @@ export function init(formDefaults, render) {
 function getConfig(formData) {
 	const {
 		config,
-		tokenValue, tokenType, tokenIssuer,
+		tokenValue: value,
+		tokenType: type,
+		issuer,
```

Comment:
formData is the form fields. they should have a tokenValue/tokenType from the input[name] attrs.